app/src/main/java/com/yacgroup/yacguide/CountryActivity.kt

```diff
@@ -67,4 +78,26 @@ class CountryActivity : UpdatableTableActivity() {
     override fun getLayoutId(): Int {
         return R.layout.activity_country
     }
+
+    /*
+     * Check whether the "What's New" dialog is shown or not. This is decided by comparing
+     * the version stored in the settings with the current app version.
+     */
+    private fun _init() {
+        val settingsKeyVerCode = "version_code"
+        var curVerCode = 0
```

Comment:
To keep code clean and secure, try to avoid non-const. variables wherever possible. Simple solution here would be:
```
val currVersionCode = try {
packageManager.getPackageInfo(packageName, 0).versionCode
} catch (e: Exception) {
0
}
```